test_bot/cogs/misc.py

```diff
@@ -29,15 +29,17 @@ async def attachment_desc(self, inter: disnake.AppCmdInter, desc: str = "test")
         await inter.response.send_message(file=self._get_file(desc))
 
     @commands.slash_command()
-    async def attachment_desc_edit(self, inter: disnake.AppCmdInter, desc: str = "test") -> None:
+    async def attachment_desc_edit(
+        self, inter: disnake.AppCmdInter[commands.Bot], desc: str = "test"
```

Comment:
It would be cool to have this changed in all of `test_bot/`, instead of this one place only